src/azul/plugins/metadata/anvil/indexer/transform.py

```diff
@@ -498,6 +498,7 @@ def _duos_types(cls) -> FieldTypes:
         return {
             'document_id': null_str,
             'description': null_str,
+            'duos_id': null_str,
```

Comment:
We should also change azul.plugins.metadata.anvil.indexer.transform.SingletonTransformer._is_duos to use `duos_id` instead of `description`.